docs/docs/axes/cartesian/time.md

```diff
@@ -71,21 +71,21 @@ var chart = new Chart(ctx, {
 
 ### Display Formats
 
-The following display formats are used to configure how different time units are formed into strings for the axis tick marks. See [Moment.js](https://momentjs.com/docs/#/displaying/format/) for the allowable format strings.
+You may specify a map of display formats with a key for for each unit:
 
-Name | Default | Example
---- | --- | ---
-`millisecond` | `'h:mm:ss.SSS a'` | `'11:20:01.123 AM'`
-`second` | `'h:mm:ss a'` | `'11:20:01 AM'`
-`minute` | `'h:mm a'` | `'11:20 AM'`
-`hour` | `'hA'` | `'11AM'`
-`day` | `'MMM D'` | `'Sep 4'`
-`week` | `'ll'` | `'Sep 4 2015'`
-`month` | `'MMM YYYY'` | `'Sep 2015'`
-`quarter` | `'[Q]Q - YYYY'` | `'Q3 - 2015'`
-`year` | `'YYYY'` | `'2015'`
+* `millisecond`
+* `second`
+* `minute`
+* `hour`
+* `day`
+* `week`
+* `month`
+* `quarter`
+* `year`
 
-For example, to set the display format for the `quarter` unit to show the month and year, the following config would be passed to the chart constructor.
+The format string used as a key depends on the date adapter you chose to use.
```

Comment:
The format string is actually value and not key?

Reply:
fixed